### PR TITLE
updates README - styles

### PR DESCRIPTION
**Pull request by 311289189**
I want to add this clarification about the `style` prop. I wasted a bit of time because I had to assume the prop was styles, before finally realising it is `style` so I've enhanced the example with a real use case

Fixes #[issue number].

Changes proposed:
-
-
-

Upgrade Path (for changed or removed APIs):

Acceptance Checklist:
- [ ] All commits have been squashed to one.
- [ ] The commit message follows the guidelines in `CONTRIBUTING.md`.
- [ ] Documentation (README.md) and examples have been updated as needed.
- [ ] If this is a code change, a spec testing the functionality has been added.
- [ ] If the commit message has [changed] or [removed], there is an upgrade path above.

### CODE REVIEW

**Comment by diasbruno** [comment]
Thanks, @311289189. Can you please add `[chore]` at the beginning of the commit message?

**Comment by 311289189** [comment]
@diasbruno done!

**Comment by diasbruno** [comment]
Ah, can you also copy the example to the `doc/style` page?

**Comment by 311289189** [comment]
let me know if it needs changes

**Comment by diasbruno** [comment]
Thank you so much for having a look on this docs.

File: docs/styles/README.md [hunk 1]
```diff
@@ -3,28 +3,28 @@
 Styles passed into the Modal via the `style` prop are merged with the defaults.  The default styles are:
 
 ```js
```
Comment (diasbruno) [inline docs/styles/README.md]: I think this is the last thing to get this merged. 

You can use change the sintax to `jsx` and add the component `<Modal ... />`.

Comment (311289189) [reply docs/styles/README.md]: ah ok, yep

**Comment by diasbruno** [comment]
Tomorrow, I'll finish to review the other PRs and make the release `3.1.12` and the docs. Thank you!

**Comment by diasbruno** [comment]
Published new documentation version.

**Pull request merged**